recipes/delocate/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.7.4" %}
+
+package:
+  name: delocate
+  version: {{ version }}
+
+source:
+  fn: delocate-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/delocate/delocate-{{ version }}.tar.gz
+  sha256: 404819b9f4536903c432e9d99c3d5993ba267f86d73cab51d76cc0d6e04dfd55
+
+build:
+  number: 0
+  skip: true  # [not osx]
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
May want to add `--no-deps` here. Believe this was fixed in recent versions of `conda-build`, but recall we had pinned an older version of `conda-build` in a few places including `staged-recipes`. So they may not have the fix.

Reply:
ok, but i just followed https://conda-forge.org/docs/meta.html#use-pip, so ...

Reply:
You're absolutely right. We have had some minor setbacks on this front. We would like to get the syntax to this level of simplicity though. Sorry for the trouble.